admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -78,25 +79,47 @@ object DatabaseTablesDAO {
       queryParameters: QueryParameters,
       settings: TableSettings
   )(implicit conn: Connection): List[TableRow] = {
+    val datePattern = "([12]\\d{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])".r
```

Comment:
When adding regexps, it is useful to comment working examples, I'm also wondering whether this is the place to check this

Alternatively, create a helper object with tests for parsing dates